### DISCUSSION TITLE
Why can't Phantom run a Secp256k1 instruction?

### DISCUSSION THREAD

**Discussion by IvanZen** [phantom/docs#159] [Help & Support]
### Summary

I can't run a Secp256k1 instruction with Phantom since it shows a error with `WalletSendTransactionError: Unexpected error` everytime, but I can do this with Solflare wallet. What should I do to run it?

### Steps to Reproduce

Create and run a Secp256k1 inscrtuction with native Secp256k1 program and @solana/web3.js

### Phantom Version

23.19.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hi @IvanZen,

Would you mind providing your code and use case please?
